website/core/ContactForm/country-codes.js

```diff
@@ -58,6 +69,10 @@ const countryCodes = [
     "country": "Austria",
     "abbreviation": "AT"
   },
+ {
```

Comment:
This is the only one that was missing. The other 3 are duplicated. This should all be fixed with https://github.com/libra/website/pull/48 though.